languages/csharp/concepts/enums/links.json

```
@@ -1 +1,18 @@
-[]
+[
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum",
+    "description": "enumeration types"
+  },
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/api/system.enum.parse?view=net-5.0",
+    "description": "Enum.Parse"
+  },
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/api/system.enum.tryparse?view=net-5.0",
+    "description": "Enum.TryParse"
```

Maybe make this a bit more friendly to read?

```suggestion
    "description": "Parsing enums"
```